Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -1611,7 +1611,9 @@ namespace Remotion.BocAutoCompleteReferenceValue
             this.element.style.position = 'fixed';
 
             LayoutUtility.Hide(this.element);
-            this.input.closest('div, td, th, body')!.appendChild(this.element);
+
+            this.input.closest('div, td, th, body, span.bocListEditableCell > span.control')!.appendChild(this.element);
+
 
             if (this.options.combobox.getAttribute('aria-labelledby') !== null) {
```

Comment:
Got some extra whitespace here. 
also since we we will now change the develop-version to 4.0, this non-breaking change will get rebased onto v3.10-branch (new branch name with -v3.10 suffix) and we do RM-1492 instead for develop, which will block RM-8656.